qcengine/programs/tests/test_turbomole.py

```diff
@@ -0,0 +1,94 @@
+import numpy as np
+import pytest
+
+import qcelemental
+from qcelemental.testing import compare_values
+import qcengine
+from qcengine import testing
+
+
+@pytest.fixture
+def h2o():
+    mol = qcelemental.models.Molecule.from_data("""
+            O 0.000000000000     0.000000000000    -0.068516245955
+            H 0.000000000000    -0.790689888800     0.543701278274
+            H 0.000000000000     0.790689888800     0.543701278274
+    """)
+    return mol
+
+
+@pytest.mark.parametrize(
+    "method, keywords, ref_energy",
+    [
+        pytest.param('hf', {}, -75.95536954370, marks=testing.using_turbomole),
+        pytest.param('pbe0', {"grid": "m5"}, -76.27371135900, marks=testing.using_turbomole),
+        pytest.param('ricc2', {}, -76.1603807755, marks=testing.using_turbomole),
+        pytest.param('rimp2', {}, -76.1593614075, marks=testing.using_turbomole),
+    ])  # yapf: disable
+def test_turbomole_energy(method, keywords, ref_energy, h2o):
+    resi = {
+        "molecule": h2o,
+        "driver": "energy",
+        "model": {
+            "method": method,
+            "basis": "def2-SVP",
+        },
+        "keywords": keywords,
+    }
+
+    res = qcengine.compute(resi, "turbomole", raise_error=True, return_dict=True)
+
+    assert res["driver"] == "energy"
+    assert res["success"] is True
+
+    assert compare_values(ref_energy, res["return_result"])
+
+
+@pytest.mark.parametrize(
+    "method, keywords, ref_norm",
+    [
+        pytest.param('hf', {}, 0.099340, marks=testing.using_turbomole),
+        pytest.param('pbe0', {"grid": "m5"}, 0.060631, marks=testing.using_turbomole),
+        pytest.param('ricc2', {}, 0.059378, marks=testing.using_turbomole),
+        pytest.param('rimp2', {}, 0.061576, marks=testing.using_turbomole),
+    ])  # yapf: disable
+def test_turbomole_gradient(method, keywords, ref_norm, h2o):
+    resi = {
+        "molecule": h2o,
+        "driver": "gradient",
+        "model": {
+            "method": method,
+            "basis": "def2-SVP",
+        },
+        "keywords": keywords,
+    }
+
+    res = qcengine.compute(resi, "turbomole", raise_error=True)
+
+    assert res.driver == "gradient"
+    assert res.success is True
+
+    grad = res.return_result
+    grad_norm = np.linalg.norm(grad)
+    assert compare_values(ref_norm, grad_norm)
+
+
+def test_turbomole_ri_dsp(h2o):
```

Comment:
This needs a `testing.using_turbomole` decorator.

Reply:
fixed ind bf18f59